src/components/CompareResults/RevisionRow.tsx

```diff
@@ -322,16 +339,20 @@ function RevisionRow(props: RevisionRowProps) {
             </div>
           </Tooltip>
         </div>
-        <div className='base-value cell' role='cell'>
-          {' '}
-          {formatNumber(baseAvgValue)} {baseUnit}{' '}
+        <div className='browser-name cell' role='cell'>
+          {formatNumber(baseAvgValue)} {baseUnit}
+          {getBrowserDisplay(baseApp, newApp, expanded) && (
+            <span style={{ fontSize: '11.5px' }}>({baseApp})</span>
+          )}
         </div>
         <div className='comparison-sign cell' role='cell'>
           {determineSign(baseAvgValue, newAvgValue)}
         </div>
-        <div className='new-value cell' role='cell'>
-          {' '}
+        <div className='browser-name cell' role='cell'>
           {formatNumber(newAvgValue)} {newUnit}
+          {getBrowserDisplay(baseApp, newApp, expanded) && (
+            <span style={{ fontSize: '11.5px' }}>({newApp})</span>
+          )}
```

Comment:
Once you added the `fontSizeRaw` to `browser-name`, please delete these inline styles.